extensions/MozChangeField/Extension.pm

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::Extension::MozChangeField;
+
+use 5.10.1;
+use strict;
+use warnings;
+
+use base qw(Bugzilla::Extension);
+
+use Bugzilla::Constants;
+use Bugzilla::Logging;
+
+use File::Basename qw(basename);
+use Module::Runtime qw(require_module);
+use Try::Tiny;
+
+our $VERSION = '0.1';
+
```

Comment:
I'd like to see some inline doc here describing the expectations for `MozChangeField` submodules, along with the expected results and their meanings.